lib/matplotlib/tests/test_backend_pdf.py

```diff
@@ -191,3 +192,23 @@ def psfont(*args, **kwargs):
     ax.text(0.5, 0.5, 'hello')
     with tempfile.TemporaryFile() as tmpfile, pytest.raises(ValueError):
         fig.savefig(tmpfile, format='pdf')
+
+
+@needs_tex
+def test_failing_latex():
```

Comment:
You can use the `tmpdir` fixture to get a temporary directory here; shouldn't need `tempfile`.